alexandria/core/views.py

```diff
@@ -36,17 +38,18 @@
     MarkFilterSet,
     TagFilterSet,
 )
-from .storage_clients import client
-from .utils import create_thumbnail, get_checksum, get_file
+from .utils import create_thumbnail, get_checksum
 
 
 class PermissionViewMixin:
     def destroy(self, request, *args, **kwargs):
         self.queryset.model.check_permissions(request)
         instance = self.get_object()
         instance.check_object_permissions(request)
-        # we do not call `super()` in order to not fetch the object twice.
-        self.perform_destroy(instance)
+        # ~~we do not call `super()` in order to not fetch the object twice.~~
+        # NOTE: we call `super()` in order to not circumvent the storage
+        # backend.
```

Comment:
I don't think we should keep old comments.. Just drop it

```suggestion
        # NOTE: we call `super()` in order to not circumvent the storage
        # backend.
```

Reply:
done